### PR TITLE
fix wrong example of Gather-11/Gather-1 documentation

### PR DESCRIPTION
**Pull request by Tee0125**
Signed-off-by: Tee Jung <tee.ty.jung@openedges.com>

### Description

Gather-1/Gather-11 example in documentation is not matched with onnxruntime's result

Fixing wrong documentation of Gather-1/Gather-11's behavior when axis=1
 
### Motivation and Context

onnxruntime's result of Gather-13 is matched with document, but results of Gather-1/Gather-11 are not matched with document so I fixed it

https://github.com/onnx/onnx/issues/4809

### CODE REVIEW

**Comment by daquexian** [comment]
Thanks for your contribution!

docs/Changelog.md is auto-generated. Please update the content of `Gather_ver11_doc` and `Gather_ver1_doc` in onnx/defs/tensor/old.cc instead.

**Review by jcwchen — APPROVED** [review]
LGTM. Thank you for fixing!

**Comment by gramalingam** [comment]
After changing the source, need to run `python onnx\defs\gen_doc.py` to generate the documentation and include those changes in the PR

**Comment by Tee0125** [comment]
ok, I pushed fresh new docs

File: onnx/defs/tensor/old.cc [hunk 1]
```diff
@@ -1631,11 +1631,9 @@ output[i_{0}, ..., i_{q-1}, j_{0}, ..., j_{r-2}] = input[j_{0}, k, j_{1}, ..., j
   ]
   axis = 1,
   output = [
-      [
-          [1.0, 1.9],
-          [2.3, 3.9],
-          [4.5, 5.9],
-      ],
+      [[1.0, 1.9]],
```
Comment (gramalingam) [inline onnx/defs/tensor/old.cc:1634]: Thanks for the fix. I think the line 1621 also needs to be updated (as in Gather-13)? May be same for Gather-1 also down below?

Comment (Tee0125) [reply onnx/defs/tensor/old.cc:1634]: line 1621 is updated as Gather-13 and Gather-1's doc has only input/output definition in example so, I think, no update is needed.

**Review by gramalingam — APPROVED** [review]

**Comment by gramalingam** [comment]
Great, thanks very much for the PR! For some reason, it doesn't allow me to update to current main branch, you may need to do it.

**Pull request merged**